### ISSUE TITLE
Different texts for Labels and legends

### ISSUE THREAD

**Issue by yancorrea1995** [greghub/funnel-graph-js#5]
Would it be possible to add this feature?

**Add a setting to set legends (subtitles) and Labels separately. One field for each object.**

Thanks!

**Comment by greghub** [comment]
Hi @yancorrea1995 , thanks for you interest!
could you please clarify by "setting separately"?
Currently, it is possible to set labels and sub labels (legends):

```js
data: {
    labels: ['Impressions', 'Add To Cart', 'Buy'],
    subLabels: ['Direct', 'Social Media', 'Ads'],
    ...
}
```

What would be wish to set the values?

**Comment by yancorrea1995** [comment]
Hi Greg!

During mouse hover, add the possibility of customizing the text, so that it is different from the legend in the bottom of funnel.
![export](https://user-images.githubusercontent.com/39925733/57468050-1b2bf700-725a-11e9-87c9-0027de62b446.png)

**Comment by yancorrea1995** [comment]
And, if possible, add an option to hide the zero values in the mouse hover. Example (hide "Idea 02: 0" because the "0" is not a necessary information. That will make the Funnel more clean and easy to view.

**Comment by lordplagus02** [comment]
Agreed with removing zeroes for usability, but having to adapt the data to remove subsets with no values made for some efficient backend coding 😛

**Comment by yancorrea1995** [comment]
Don't need to remove the values, just hide. Who to use chooses whether or not to hide with a flag (option) in the settings of the funnel.

**Comment by maitsa** [comment]
I made a workaround to hide empty sublabel values. The last "if" hide also results with only 1 value, to avoid redundancy with main label.

    $('div.svg-funnel-js ul.segment-percentage__list').each(function (i, list) {
        list = $(list);
        var listItems = list.find('span.percentage__list-label');
        listItems.each(function (j, elem) {
            elem = $(elem);
            if (elem.text().startsWith("0")) {
                elem.closest('li').addClass('hidden');
            }
        });

        if (list.find('li:not(.hidden)').length <= 1) {
            list.closest('.label__segment-percentages').addClass('hidden');
        }        
    });
